curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -533,8 +543,10 @@ bool FileCacheManager::ReadKVRequestFromS3(const std::string &name,
     }
 
     if (s3ClientAdaptor_->s3Metric_) {
-        s3ClientAdaptor_->CollectMetrics(
-            &s3ClientAdaptor_->s3Metric_->adaptorReadS3, length, start);
+        metric::CollectMetrics(&s3ClientAdaptor_->s3Metric_->adaptorReadS3,
+                               length, butil::cpuwide_time_us() - start);
+        metric::CollectMetrics(&s3ClientAdaptor_->s3Metric_->readFromS3, length,
+                               butil::cpuwide_time_us() - start);
```

Comment:
It's used in more than one place, For example in `DiskCacheManagerImpl::Read`  , so I think you can search `Download` in all file.

ditto other metric

Reply:
> It's used in more than one place, For example in `DiskCacheManagerImpl::Read` , so I think you can search `Download` in all file.
> 
> ditto other metric

fix